component.yaml

```
@@ -18,14 +18,19 @@ inputs:
   - key: "AC_RN_TEST_COMMAND_ARGS"
     isRequired: false
     title: Jest Extra Arguments
-    description: Specify additional arguments for running the Jest command. These arguments will be added to the end of the command `jest` which will be used by default. You can add extra arguments, such as `--debug --colors`, without affecting the default ones. For more information, see the Jest [CLI options](https://jestjs.io/docs/cli#options).
+    description: Specify additional arguments for running the Jest command. These arguments will be added to the end of the command `jest --coverage --coverageDirectory=coverage --coverageReporters=lcov` which will be used by default. You can add extra arguments, such as `--debug --colors`, without affecting the default ones. For more information, see the Jest [CLI options](https://jestjs.io/docs/cli#options).
```

If you added this parameter as defaultValue, you don't need "such as":
```suggestion
    description: Specify additional arguments for running the Jest command. These arguments will be added to the end of the command `jest --coverage --coverageDirectory=coverage --coverageReporters=lcov` which will be used by default. You can add extra arguments without affecting the default ones. For more information, see the Jest [CLI options](https://jestjs.io/docs/cli#options).
```